yt/frontends/cf_radial/fields.py

```diff
@@ -0,0 +1,36 @@
+"""
+CF-radial-specific fields
+
+
+
+"""
+from typing import List
+
+from yt.fields.field_info_container import FieldInfoContainer
+
+
+class CFRadialFieldInfo(FieldInfoContainer):
+    known_other_fields = ()  # fields are set dynamically
+    known_particle_fields = ()
+    units_to_ignore = ("dBz", "dBZ", "ratio")  # set as nondimensional if found
```

Comment:
why ?

Reply:
If I remember right, it's just that these units are complex (e.g., https://en.wikipedia.org/wiki/DBZ_(meteorology) ) and it wasn't obvious how to handle them... but I don't quite remember why the `units_to_ignore` attribute was added as opposed to simply adding them as dimensionless fields in `known_other_fields`. Might have been just to have an explicit record of how the fields were not being handled well.  

I'm pretty sure they are in fact all dimensionless units so I think we could define them in the `CFRadialFieldInfo.__init__()` method (e.g., `self.ds.define_unit("dBZ", 1*unyt.dimensionless)`). But I also remember this being an issue with the nc4_cm1 frontend (and it may have actually been handled differently), so I think it'd be better to improve this together with nc4_cm1 in a followup (I can open an issue to track it).

Reply:
Sounds good. Thank you !

Reply:
opened! https://github.com/yt-project/yt/issues/3972 I'll let you resolve the thread